magento/magento2
# [MFTF] Refactoring of AddOutOfStockProductToCompareListTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Create an out of stock product
2. Disable the "Display Out of Stock Products" option
3. Open the PDP 
4. Check if the "Add to compare" link isn't available
5. Enable the "Display Out of Stock Products" option
6. Open the PDP
7. Check if the "Add to compare" link is available
8. Add product to the Comparison list
9. Go to the Comparison list and check if the product is displayed there
10. Clear the Comparison list
11. Go to the product's category
12. Hover the product
13. Add product to the Comparison list
14. Go to  theComparison list and check if the product is displayed there


### Resolved issues:
1. [x] resolves magento/magento2#31131: [MFTF] Refactoring of AddOutOfStockProductToCompareListTest
